examples/asynccopy/asynccopy.cpp

```diff
@@ -449,15 +452,10 @@ int main(int argc,char * * argv)
                     X = img_x + kernelSize - 1;
                 if ( Y > img_y + kernelSize - 1 )
                     Y = img_y + kernelSize - 1;
-                host( y, x )( px::R() ) = Element((*image)
-                    [ Y - kernelSize ]
-                    [ X - kernelSize ].red) / 255.;
-                host( y, x )( px::G() ) = Element((*image)
-                    [ Y - kernelSize ]
-                    [ X - kernelSize ].blue) / 255.;
-                host( y, x )( px::B() ) = Element((*image)
-                    [ Y - kernelSize ]
-                    [ X - kernelSize ].green) / 255.;
+                const auto* pixel = &image[(Y - kernelSize) * img_x + X - kernelSize * 3];
```

Comment:
I think the pixel calculation is wrong.
* `(Y - kernelSize) * img_x` should be `(Y - kernelSize) * img_x * 3` as each scanline is three times as big as `img_x`.
* `X - kernelSize * 3` should  be `(X - kernelSize) * 3` for the same reason.
At least if stb_image reads images as `rgbrgbrgbrgb`. But for `rrrrggggbbbb` the code would be wrong, too. :wink:

Reply:
Well observed! I have have missed that when I pulled the changes for this branch together!

Reply:
Should be fixed